src/SingleProject/Resizetizer/src/nuget/buildTransitive/Microsoft.Maui.Resizetizer.After.targets

```diff
@@ -386,10 +386,25 @@
             MauiSplashScreen="@(MauiSplashScreen)"
             InputsFile="$(_MauiSplashInputsFile)"
         />
+
+        <!-- delete the res.zip cache file as things have changed. -->
+        <Delete
+            Condition="'$(_ResizetizerIsAndroidApp)' == 'True' and Exists ('$(_MauiIntermediateSplashScreen)..\sp.zip')"
+            Files="$(_MauiIntermediateSplashScreen)..\sp.zip"
+        />
+
+        <ZipDirectory
+            Condition="'$(_ResizetizerIsAndroidApp)' == 'True' and Exists ('$(_MauiIntermediateSplashScreen)')"
+            SourceDirectory="$(_MauiIntermediateSplashScreen)"
+            DestinationFile="$(_MauiIntermediateSplashScreen)..\sp.zip"
+        />
+
         <ItemGroup Condition="'$(_ResizetizerIsAndroidApp)' == 'True' and '$(_MauiHasSplashScreens)' == 'true'">
             <_MauiSplashAssets Include="$(_MauiIntermediateSplashScreen)**\*" />
             <LibraryResourceDirectories Condition="Exists('$(_MauiIntermediateSplashScreen)')" Include="$(_MauiIntermediateSplashScreen)">
                 <StampFile>$(_ResizetizerStampFile)</StampFile>
+                <AndroidSkipResourceProcessing>true</AndroidSkipResourceProcessing>
+                <ResourceDirectoryArchive>$(_MauiIntermediateSplashScreen)..\sp.zip</ResourceDirectoryArchive>
```

Comment:
What is this new fangled magic?

Reply:
So `aapt2` supports both `flat` and `flata` files. The former is for single files, the later is a zip file which contains the `flat` files. The `flata` is created from a `zip` file which contains all the resources. It helps to speed up compilation because `aapt2` can operate on the `flata` archive directly without having to uncompress it. We use it for aar files and for library projects which contain allot of resources which will no change very often. 

That applies to the spalshscreens. I'm not sure it will apply to the main `r` folder though as I think that can contain custom android `layout` files which might need to be fixed up later in the android build process.

Reply:
I backed these changes out. They should probably be in a seperate PR.